FF1Lib/ItemLists.cs

```diff
@@ -35,6 +35,13 @@ public static class ItemLists
 			Item.Ribbon, Item.Ribbon, Item.OpalGauntlets, Item.OpalGauntlets,
 		};
 
+		public static readonly IReadOnlyList<Item> TypeWeaknessWeapon =

```

Comment:
For compatibility with weaponizer, you can simply cycle through all weapons and check the bytes for elemental and type weakness (if it's excalibur, it's $FF, if there's none it's $00)

`
var weapondata = new Weapon(weaponid, rom);
if weapon.ElementalWeakness ­­> $00 & < $FF
if weapon.TypeWeakness ­­> $00 & < $FF
  then it's a weapon that does extra damage, add it to the list
`

Reply:
Good call wildham, I've added this kind of logic. It will now be useable with weaponizer.